src/main/java/org/broadinstitute/hellbender/tools/spark/sv/Kmer.java

```diff
@@ -0,0 +1,177 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import java.io.Serializable;
+
+/**
+ * An immutable Kmer.
+ * K must be between 1 and 63 (but it's silly to use this class for K < 33).
+ * Canonicalization is unimplemented for even K.
+ *
+ * Created by tsharpe on 12/10/15.
+ */
+public final class Kmer implements Comparable<Kmer>, Serializable
+{
+    private static final long serialVersionUID = 1L;
+
+    // these are treated as K-bit unsigned integers
+    private final long v1; // most significant K bits
```

Comment:
these could be more clearly named as "high" and "low" I think, but I don't feel strongly about that...